cvat-canvas/README.md

```diff
@@ -0,0 +1,93 @@
+## Module
+
+- Written on typescript
+- Contains the class ```Canvas```
+
+## Creation
+Canvas is created by using constructor:
+
+```js
+    const canvas = new Canvas();
+```
+
+- Canvas has transparent background
+
+Canvas itself handles:
+- Shape context menu (PKM)
+- Image moving (mousedrag)
+- Image resizing (mousewheel)
+- Image fit (dblclick)
+- Remove point (PKM)
+- Polyshape editing (Shift + LKM)
+
+## API
+### Methods
+
+All methods are sync.
+
+```js
+    html() => canvas HTML element
+    setup(const FrameData: frameData, [{
+        state: ObjectState,
+        appearance: {   // all these fields are optional
+            borderColor: 'color',
+            fillColor: 'color',
+            fillOpacity: 50,
+            selectedFillOpacity: 50,
+        }
+    }]) => undefined
+
+    activate(const number: clientID, const number: attributeID = null) => undefined // select if can't activate
+    rotate(const number: degrees = 90) => undefined
```

Comment:
Does it support 30 degrees rotation?

Reply:
We can support it, but I am not really sure we need it. 
Interface can be ```rotate(const Rotation: direction = Rotation.CLOCKWISE) // Where Rotation is enum```